composer.json

```diff
@@ -1,33 +1,41 @@
 {
-    "name": "drift/skeleton",
-    "description": "Skeleton repository ofr DriftPHP",
-    "type": "project",
-    "license": "MIT",
-    "authors": [
-        {
-            "name": "Marc Morera",
-            "email": "yuhu@mmoreram.com"
-        }
-    ],
-    "require": {
-        "php": "^7.3",
-        "symfony/console": "^5.0.0",
-        "symfony/framework-bundle": "^5.0.0",
-        "symfony/dotenv": "^5.0.0",
-        "symfony/stopwatch": "^5.0.0",
-        "symfony/debug": "^4.4.0",
-
-        "drift/http-kernel": "0.1.*, >=0.1.7",
-        "drift/server": "0.1.*, >=0.1.7",
-        "drift/react-functions": "0.1.*",
-        "drift/event-loop-utils": "0.1.*"
-    },
-    "autoload": {
-        "psr-4": {
-            "App\\": "src/",
-            "Domain\\": "src/Domain",
-            "Infrastructure\\": "src",
-            "Drift\\": "Drift/"
-        }
+  "name": "drift/skeleton",
+  "description": "Skeleton repository ofr DriftPHP",
+  "type": "project",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Marc Morera",
+      "email": "yuhu@mmoreram.com"
     }
-}
\ No newline at end of file
+  ],
+  "require": {
+    "php": "^7.3",
+    "symfony/console": "^5.0.0",
+    "symfony/framework-bundle": "^5.0.0",
+    "symfony/dotenv": "^5.0.0",
+    "symfony/stopwatch": "^5.0.0",
+    "symfony/debug": "^4.4.0",
+    "drift/http-kernel": "0.1.*, >=0.1.7",
+    "drift/server": "0.1.*, >=0.1.7",
+    "drift/react-functions": "0.1.*",
+    "drift/event-loop-utils": "0.1.*",
+    "symfony/flex": "^1.8"
```

Comment:
TBH, I don't like `flex`. It introduces an insane dependency to Symfony environment in terms of the project, and DriftPHP should only use some Symfony components, but be completely detached from the project itself.

I've been working without Flex for the last years, and TBH, I didn't even notice it at all.

Reply:
Flex follows the SoC concept on the config layer and is the standard way of creating Symfony apps since 5.x, so you should adapt it.